packages/terra-site/src/examples/dialog/DialogDefault.jsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-alert */
 import React from 'react';
-import Dialog from 'terra-dialog';
+import Dialog from 'terra-dialog/src/Dialog';
```

Comment:
Shouldn't this be lib?

Reply:
This will be removed. Just using the src directory for live reloading.

Reply:
Let's get this switched over and then this should be good to merge down @saedar